packages/node_modules/@ciscospark/react-redux-spark/src/component.js

```diff
@@ -24,10 +26,13 @@ class SparkComponent extends Component {
 
     if (!spark) {
       spark = createSpark(accessToken);
+      metrics.sendLoadStart(spark);
+      metrics.sendElapsedTimeMetric(`sparkObject:loaded`, spark);
       props.storeSparkInstance(spark);
     }
 
     spark.listenToAndRun(spark, `change:canAuthorize`, () => {
+      metrics.sendElapsedTimeMetric(`sparkObject:authenticated`, spark);
```

Comment:
ooo i think we should actually capture a little more here... if this hits multiple times during a single session then we can actually see that there are multiple connects.